.github/workflows/cove-ocds-live-tests.yml

```
@@ -0,0 +1,34 @@
+name: CI
```

We use "CI" when doing continuous integration on the repo itself.

```suggestion
name: OCDS Data Review Tool
```